src/hotspot/share/oops/fieldInfo.hpp

```diff
@@ -47,19 +47,21 @@ class FieldInfo {
   // as an array of 6 shorts.
 
 #define FIELDINFO_TAG_SIZE             2
-#define FIELDINFO_TAG_BLANK            0
-#define FIELDINFO_TAG_OFFSET           1
-#define FIELDINFO_TAG_TYPE_PLAIN       2
-#define FIELDINFO_TAG_TYPE_CONTENDED   3
-#define FIELDINFO_TAG_MASK             3
+#define FIELDINFO_TAG_OFFSET           1 << 0
+#define FIELDINFO_TAG_CONTENDED        1 << 1
 
   // Packed field has the tag, and can be either of:
   //    hi bits <--------------------------- lo bits
   //   |---------high---------|---------low---------|
-  //    ..........................................00  - blank
+  //    ..........................................CO
+  //    ..........................................00  - non-contended field
+  //    [--contention_group--]....................10  - contended field with contention group
   //    [------------------offset----------------]01  - real field offset
-  //    ......................[-------type-------]10  - plain field with type
-  //    [--contention_group--][-------type-------]11  - contended field with type and contention group
+
+  // Bit O indicates if the packed field contains an offset (O=1) or not (O=1)
```

Comment:
Hi Fred, should this comment say  "... or not (0=0)   ?

Reply:
You're right. I've fixed the comment (and the line below which had the same issue).

Fred